### PR TITLE
Remove info now part of docs

### PR DESCRIPTION
**Pull request by beatngu13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,6 @@ Commands:
   help        Displays help information about the specified command.
 ```
 
-## Download and Installation
```
Comment (roesslerj) [inline README.md]: I like this, but at least we should put a pointer (direct link to the correct site in the docs) instead...

Comment (beatngu13) [reply README.md]: I've added a wiki tab with a link to our docs, just like in recheck and recheck-web: https://github.com/retest/recheck.cli/wiki.

Comment (roesslerj) [reply README.md]: I personally feel that this does not suffice. People are lazy and probably won't find this. I feel this is such an important information, that it should be prominently displayed. Removing it from the readme is already critical, but the link should definitely be there. Also ... you want the repo to be as self-contained as possible, not relying on the github wiki to access a simple link.

Comment (beatngu13) [reply README.md]: Then remove the wiki from all repos and simply link to the docs from a "Documentation" section inside the `README.md`?

Comment (roesslerj) [reply README.md]: We can do this either way. I just feel that the "how to install that piece of software" should be directly described or at least linked from the readme. So when we remove that info in this PR, we should put a link not to the documentation in general, but to the part where installation is described...

Comment (beatngu13) [reply README.md]: OK, added a link.

**Pull request merged**